main.tf

```diff
@@ -1,3 +1,10 @@
+data "aws_ssoadmin_instances" "this" {}
+data "aws_caller_identity" "current" {}
+
+locals {
+  instance_arn = one(data.aws_ssoadmin_instances.this.arns)
```

Comment:
It looks like `one()` will error if there is more than zero or one elements in the list and return null if the list is empty; is that desired here? (Compared to `tolist(...)[0]` which will I think will fail if the list is empty and always return the first element otherwise, but doesn't care about how many elements there are.)

Reply:
Ah interesting. Well, theoretically there is exactly one SSO Admin Instance ARN even tho AWS has decided in its API to return a list. I had seen `one` used in some example somewhere, but looking at the Terraform Registry docs it looks like they use `tolist(...)[0]` everywhere 🤔

Reply:
I'll follow the registry convention and use `tolist`